docs/_sass/_defaults.scss

```diff
@@ -105,38 +105,48 @@ h4 {
 
 .note {
   &:before {
-    content: 'Note: ';
+	content: 'Note: ';
+	font-weight: 800;
   }
+
   background: #F3FCFD;
   border-left: 40px solid #269D91;
-  margin-bottom: 10px;
-  padding: 10px;
+  border-radius: 0.3rem;
+  margin: 3rem 0;
+  padding: 1.5rem 2rem;
 }
+
 .change-role {
-  background: pink;
-  border: 3px solid pink;
-  display: flex;
-  margin-bottom: 10px;
-  .character-icon { flex-grow: 1; }
+	background: pink;
```

Comment:
we want to keep our css codebase uniform. The tabbing is a bit off. Would you be able to keep the tabs 2 spaces instead of 4 in all places css has been added?

Reply:
I took a look at this, there were a few lines off, but it looks like the double spacing was due to nesting w/in the scss, but that's ok. 
![Screen Shot 2019-03-19 at 2 24 09 PM](https://user-images.githubusercontent.com/123787/54631660-d7372580-4a52-11e9-8b1d-8315121fa294.png)

Reply:
@typaulhus this is probably bc of the settings in your IDE. if you look at the raw code, there are too many spaces on these lines - check out the [files view](https://github.com/cyberark/secretless-broker/pull/654/files) to see the diff yourself.

Reply:
I think part of the problem is that this file mixes tab and 2-space indenting - it would be good to standardize it.

Reply:
was this fixed? I tried on my own IDE that has 2 tabs but tabbing is still on wonky when pushed to GH